### PR TITLE
Rework Sponsors section

### PR DESCRIPTION
**Pull request by RobertWSaunders**
## Proposed Change

NOTE: Not mobile optimized (yet)! 

Reworks our sponsors section to be a little more clean and aligned, also adds in some partners.

### How did you do this?

Messing with CSS!

### Why are you choosing this approach?

Only way to make it look better. 😃 

### Any open questions you want to ask code reviewers?

Nope!

### List of changes:

  - Rework sponsors section
  - Add MLH as a partner
  - Add GitHub as a partner through affiliation with MLH
  - Add Bloomberg as a partner through affiliation with MLH
  - Add Stickermule as a partner through affiliation with MLH
  - Add ECE as a partner in good faith
  - Fix various links and alt descriptions

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new unit tests for my core changes (where applicable).
  - [x] I have written browser integration tests for my core changes (where application).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit c5854f237ad66b5a118a7e898b2fe1e15ea3e2b9

https://deploy-preview-209--qhacks-website.netlify.com

File: cypress/integration/Sponsors.spec.js [hunk 1]
```diff
@@ -28,26 +28,21 @@ describe("Sponsors", () => {
...
+        "https://mlh.io/"
       ];
 
       cy.get("[data-cy=sponsor-link]").each(($el, index, $list) => {
         expect($el).to.have.attr("href", sponsorURLs[index]);
       });
     });
   });
-
-  describe("Images", () => {
```
Comment (RobertWSaunders) [inline cypress/integration/Sponsors.spec.js]: This wasn't really useful so I've got rid of it for now!

File: src/components/TestimonialsBlurb.js [hunk 2]
```diff
@@ -30,7 +31,7 @@ const TestimonialsBlurb = (props) => {
     >
       <div style={{ marginRight: "64px" }}>
         <h1 style={{ marginBottom: "30px" }}>Hacker Testimonials</h1>
-        <p style={{ ...blurbStyle }}>{props.text}</p>
```
Comment (RobertWSaunders) [inline src/components/TestimonialsBlurb.js]: I'm curious why we are so inconsistent with the styling prop we use, should we not always use `css`?

File: src/components/Footer.js [hunk 3]
```diff
@@ -175,21 +176,19 @@ const Footer = () => (
...
-              css={{ ...finePrint, paddingBottom: "6px" }}
+              css={{ ...finePrint, marginBottom: "5px" }}
             >
               Queen&#39;s University | 99 University Ave, Kingston, ON
             </p>
-            <p
-              data-cy="footer-dev-plug"
-              css={{ ...finePrint, paddingBottom: "6px" }}
-            >
+            <a data-cy="footer-dev-plug" css={{ ...finePrint, fontWeight: "400" }} href="https://github.com/qhacks/qhacks-website" target="_blank" rel="noopener noreferrer">
```
Comment (RobertWSaunders) [inline src/components/Footer.js]: Yay, getting the recognition we deserve!

File: src/components/TestimonialsBlurb.js (hunk 2 above)
Comment (joeytepp) [reply src/components/TestimonialsBlurb.js]: Looks like it, i'm in favour of `css` over style

File: src/components/TestimonialsBlurbCompressed.js [hunk 4]
```diff
@@ -3,10 +3,18 @@ import React from "react";
 const TestimonialsBlurbCompressed = (props) => {
   return (
     <div
-      style={{ textAlign: "center", marginBottom: "10px", marginTop: "30px" }}
```
Comment (joeytepp) [inline src/components/TestimonialsBlurbCompressed.js]: Change this to `css`!

File: src/components/TestimonialsBlurbCompressed.js [hunk 5]
```diff
@@ -3,10 +3,18 @@ import React from "react";
...
+      style={{ textAlign: "center", marginBottom: "10px" }}
     >
       <h1 style={{ marginBottom: "16px" }}>Hacker Testimonials</h1>
```
Comment (joeytepp) [inline src/components/TestimonialsBlurbCompressed.js]: Change this to `css`!

File: src/components/TestimonialsBlurb.js [hunk 6]
```diff
@@ -30,7 +31,7 @@ const TestimonialsBlurb = (props) => {
...
+        <p css={{ ...blurbStyle }}>{props.text}</p>
         <div style={{ marginTop: "65px" }}>
```
Comment (joeytepp) [inline src/components/TestimonialsBlurb.js]: Change this to `css`!

**Pull request merged**